ui/docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
     working_dir: /w/ui
     volumes:
       - ..:/w:cached
-    command: npm start -- --host=0.0.0.0 ${ENV_FLAG}
+    command: sh -c 'npm start -- --host=0.0.0.0 ${ENV_FLAG}'
```

Comment:
What does this do [differently]? (Maybe leave a comment?)

Reply:
This got around a problem with the host not opening. @dmohs has used this workaround before, and might have more details on why that was important.

I can ask him what exactly the purpose was and throw a comment in there.